nym-node/src/node/mod.rs

```diff
@@ -700,7 +702,10 @@ impl NymNode {
         let host_details = sign_host_details(
             &self.config,
             self.x25519_sphinx_keys.public_key(),
-            self.x25519_noise_keys.public_key(),
+            &VersionedNoiseKey {
+                version: nym_noise::LATEST_NOISE_VERSION,
```

Comment:
this comment would be more appropriate in a different spot, but I just thought about it after seeing it yet again. I think it would be useful if the key serialised on disk also contained this version information because presumably `v5` key wouldn't be compatible with `v1`